genie-core/src/main/java/com/netflix/genie/core/properties/S3FileTransferProperties.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.core.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Properties for S3FileTransfer.
+ *
+ * @author mprimi
+ * @since 3.1.0
+ */
+@Getter
```

Comment:
None of the other properties class in this package is doing this, so I think i should change all of them or none. Worth doing?